components/Dialogs/src/MDCAlertController.m

```diff
@@ -402,6 +398,10 @@ - (void)setupAlertView {
   for (MDCAlertAction *action in self.actions) {
     [self addButtonToAlertViewForAction:action];
   }
+  // Explicitly overwrite the view default if true
+  if (self.mdc_adjustsFontForContentSizeCategory) {
+    self.alertView.mdc_adjustsFontForContentSizeCategory = YES;
+  }
```

Comment:
If the ordering of this invocation matters, please document it here so that future us doesn't accidentally change the order again.

Reply:
Done

Reply:
Please also clarify which properties matter with respect to order here.